.env

```diff
@@ -0,0 +1,11 @@
+AWS_REGION=us-east-1
+AWS_FUNCTION_NAME=DocsService
+AWS_HANDLER=index.handler
+AWS_MEMORY_SIZE=512
+AWS_TIMEOUT=30
+AWS_DESCRIPTION=
+AWS_RUNTIME=nodejs4.3
+EXCLUDE_GLOBS="*.env,event.json"
+PACKAGE_DIRECTORY=build
+
+ENVIRONMENT=local
```

Comment:
Do we know what `ENVIRONMENT` controls? I don't see it in the code.

Reply:
Nope, I was just hesitant to remove it